apps/api/src/domain/types/AggregateType.ts

```diff
@@ -1,7 +1,13 @@
+import { ValueType } from '../../lib/types/valueType';
 import { CategorizableType } from './CategorizableType';
 
 export const AggregateType = {
     ...CategorizableType,
```

Comment:
Will this have the user-related types here?

Reply:
Yes. There will be entries like {user: 'user', userGroup: 'group'}